packages/block-library/src/media-text/deprecated.js

```diff
@@ -208,7 +220,7 @@ const v4ToV5Supports = {
 	},
 };
 
-const v6Supports = {
+const v6ToV7Supports = {
```

Comment:
What is this for? Just adding one more attribute should require no migration or deprecation. Or does it?

Reply:
Deprecations are not my strength.

- If I remove the deprecation, place the block using trunk, and then switch to this branch (and build it, of course), I don't see a block validation error.
- The docs say that I need to add a deprecation **when changing the attributes of a static block.**
- Looking at the history, I added the deprecation _before_ adding the index.php file.
- The save file is unchanged.